test/sql/test_operators.py

```diff
@@ -98,15 +98,15 @@ def test_scalar(self, operator, compare_to):
         (operators.is_distinct_from, None),
         (operators.isnot_distinct_from, True),
         (operators.is_, True),
-        (operators.isnot, True),
+        (operators.is_not, True),
         (operators.is_, False),
-        (operators.isnot, False),
+        (operators.is_not, False),
         (operators.like_op, right_column),
         (operators.notlike_op, right_column),
         (operators.ilike_op, right_column),
         (operators.notilike_op, right_column),
         (operators.is_, right_column),
-        (operators.isnot, right_column),
+        (operators.is_not, right_column),
```

Comment:
I think here we could keep both the new operators and the old ones, at least until we do not actively warn on the old version

Reply:
you are absolutely right. GREAT CATCH.

there are some tests that should be adjusted to test for the legacy versions too